src/age/age_pool.c

```diff
@@ -146,7 +146,11 @@ isCypherQuery (Node* node, char** retstr)
 	return false; 
 }
 
-List* extractCypherFuncs(List *ParsedCyphertreeList)
+List* 
+/* Returns all the Cypher Nodes
```

Comment:
This comment style within the function doesn't meet **AGE Code Style Guide** at commenting section  
References: 
- https://age.apache.org/contribution/guide/#code-review-process